EIPS/eip-7069.md

```diff
@@ -157,11 +156,13 @@ The functionality of specifying output buffer address is removed, because it is
 
 ### Status codes
 
-Current call instructions return a boolean value to signal success: 0 means failure, 1 means success. The Solidity compiler assumed this value is a boolean and thus uses the value as branch condition to status (`if iszero(status) { /* failure */ }`). This prevents us from introducing new status codes without breaking existing contracts. At the time of the design of [EIP-211](./eip-211.md) the idea of return a specific code for revert was discussed, but ultimately abandoned for the above reason.
```

Comment:
Maybe move this to rationale or motivation?

Reply:
This is a part of Rational. I can change it to something saying that we considered also a version which does not change the meaning of 0 and 1 but the extensible version won.

Reply:
Why remove this explainer then?

Reply:
Because it's not valid any longer. We do have 3 different return values for `EXT*CALL`s and one of them means that call was reverted.

Reply:
How is it not valid? It explained why it wasn't changed in the past.

Reply:
misunderstood. reverted